modules/ROOT/pages/keyword-glossary.adoc

```diff
@@ -58,6 +58,14 @@ Typically used when modifying or importing large amounts of data.
 | Schema
 | Create a constraint that ensures all relationships with a particular type have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
 
+| xref::constraints/syntax.adoc#constraints-syntax-create-node-prop-type[CREATE CONSTRAINT [property_type\] [IF NOT EXISTS\] FOR (n:Label) REQUIRE n.property IS :: TYPE [OPTIONS {}\]]
```

Comment:
I don't think we have <> anywhere else here so it feels odd to add it. and the reason why I had it in capital letters was because it still keywords that goes in there, but we could have the lowercase letters

Reply:
maybe "{INT | ...}"?

Reply:
maybe, but we don't want to add confusion with the {} in the empty options map either (I guess the relationship pattern either looks stupid or just accepts the double use of [])